Gordon360/Models/webSQL/Models/account_profile.cs

```diff
@@ -13,7 +13,7 @@ public partial class account_profile
     {
         [Key]
         public int account_id { get; set; }
-        [StringLength(75)]
+        [StringLength(8000)]
```

Comment:
As we discussed, not sure why this is here.  Since this isn't related to the changes this branch addresses, perhaps we should remove this change from the branch.

Reply:
@amos-cha Do you know why this could have been automatically updated? I checked the webSQL database and the account_id is still varchar(75)

Reply:
pull request to solve this is ready for approval: https://github.com/gordon-cs/gordon-360-api/pull/969